README.md

```diff
@@ -88,14 +90,19 @@ For possible issues while running a node refer to our [FAQ](https://github.com/m
 
 Please read [CONTRIBUTING.md](./CONTRIBUTING.md) for details on our code of conduct, and the process for submitting pull requests to us.
 
-## Authors
-* **Valdas Petrulis** - *Lead developer, go evangelist* - [Waldz](https://github.com/Waldz)
+## Contributors
```

Comment:
Maybe this should be just a link to the https://github.com/mysteriumnetwork/node/graphs/contributors page?

Reply:
We have that link. But IMHO its good to have a list of ppl that contributed the most. For example added @chompomonim too since he contributed great deal to other repos, that are not properly reflected by this repo.